src/olympia/abuse/cinder.py

```diff
@@ -496,9 +516,13 @@ def flag_for_human_review(self, appeal=False):
                 user=core.get_user() or get_task_user(),
             )
 
-    def report(self, *args, **kwargs):
-        self.flag_for_human_review(appeal=False)
-        return super().report(*args, **kwargs)
+    def post_report(self, job):
+        if not (job.is_appeal and job.is_unresolved):
```

Comment:
yeah cinder only adds new reports to open jobs*

*(technically open and unassigned, but assignment isn't applicable here, because you can't have an open escalated job)

Reply:
https://github.com/mozilla/addons-server/pull/22401/commits/6bd6d3d3d8648a9327ab612cb94d5308c5c2e447 removes that code